chrome/content/zotero/fileInterface.js

```diff
@@ -129,7 +131,8 @@ var Zotero_File_Interface = new function() {
 	 */
 	function exportFile() {
```

Comment:
That doesn't need to happen for this patch, though. 4.0 isn't the place to make unnecessary changes, since it will just result in lots of conflicts.